Packs/UrlScan/Integrations/UrlScan/UrlScan.yml

```diff
@@ -51,6 +52,12 @@ configuration:
   type: 0
   required: false
   additionalinfo: User Agent to perform requests
+- display: API Key (only required for scanning URLs)
+  name: apikey
+  required: false
+  type: 4
+  hidden: true  
+  additionalinfo: Use api_key instead
```

Comment:
Why do you have this key here?

Reply:
Because I need to avoid breaking bc so i'm keeping the old key for those who already have a pre-configured instance

Reply:
I mean the additional details key